CMakeLists.txt

```diff
@@ -132,7 +132,7 @@ option(USE_ASAN "Enable Address Sanitizer" OFF)
 option(USE_TSAN "Enable Thread Sanitizer" OFF)
 option(USE_LIBFUZZER "Enable LibFuzzer" OFF)
 option(PYBINDINGS "Create makefiles for building python bindings" OFF)
-option(SKIP_DESKTOP "Skip building of desktop application" OFF)
```

Comment:
Can this option be reused instead of introducing a new one?

Reply:
No, not without a gargantuan refactoring.
Becuase even if you `SKIP_DESKTOP`, the `location_service.cpp` is a dependency of `storage.cpp` and `storage` classes are used throughout the `generator`, so you will pull in the Qt Positioning dependency.
Unless you do exactly what this PR is doing, and turn off the Qt Positioning dependenc explicitly, where it is pulled in.

Reply:
Hmm. OK. Done.

Reply:
Why Hmm? :)

Reply:
At first I thought it could make sense to keep a separate `SKIP_DESKTOP` and `WITHOUT_QTPOSITIONING`,
because of a theoretical possibility to use the second one without the first. And after your suggestion I have rellized, that it has very little practical use at the moment, therefore we should indeed use Occam's razor.